featuretools/tests/demo_tests/test_demo_data.py

```diff
@@ -22,8 +22,8 @@ def test_load_retail_diff():
     assert es_second['order_products'].df.shape[0] == nrows_second
 
 
-def test_mock_customer():
-    es = load_mock_customer(return_entityset=True)
+def test_mock_customer(pd_mock_customer):
+    es = pd_mock_customer
```

Comment:
Let's use the `mock_customer` fixture here so this test gets run with both the pandas and Dask versions of the fixture.